src/zenml/orchestrators/step_runner.py

```diff
@@ -522,7 +519,7 @@ def _store_output_artifacts(
             The IDs of the published output artifacts.
         """
         step_context = get_step_context()
-        output_artifacts: Dict[str, UUID] = {}
+        output_artifacts: Dict[str, ArtifactVersionResponse] = {}
```

Comment:
It looks like `ArtifactVersionResponse` is behind `TYPE_CHECKING` thus this shall fail.

Reply:
It somehow does not fail, but I don't get why 🤔
I moved it out of type checking in any case